src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -56,7 +56,7 @@ public T FishForMessage<T>(Predicate<T> isMessage, TimeSpan? max = null, string
         /// <param name="hint">The hint.</param>
         /// <param name="allMessages">If null then will be ignored. If not null then will be filled with all the messages until <paramref name="isMessage"/> returns <c>true</c></param>
         /// <returns>Returns the message that <paramref name="isMessage"/> matched</returns>
-        public T FishForMessage<T>(Predicate<T> isMessage, TimeSpan? max = null, string hint = "", List<object> allMessages = null)
+        public T FishForMessage2<T>(Predicate<T> isMessage, TimeSpan? max = null, string hint = "", List<object> allMessages = null)
```

Comment:
Maybe a better way to do this, since `FishForMessage2` isn't all that intuitive, is to just make the `allMessages` parameter not optional on the overload - place it before the optional `TimeSpan`. That would be better I think.

Reply:
Did you mean to push a new commit to resolve this message?

Reply:
No. I clicked "resolve" by mistake.

Reply:
I just pushed a new commit.